### ISSUE TITLE
Website: Getting rid of Google & Co.?

### ISSUE THREAD

**Issue by Echolon** [bascht/omemo-top#109]
What do you think?

**Comment by bascht** [comment]
@Echolon Sure! We're only using the fonts that came with the Jekyll default theme and I'm not even sure if anyone would notice if the font changed. :smile: Do you want to submit a PR for the fonts?

As of the other requests: 
![Requests](https://drop.bascht.space/5861ae64-b412-40ec-a2ed-b3e905ad0319.png)

I think we could get rid of Cloudflare, as GitHub now supports [Let's encrypt certificates](https://blog.github.com/2018-05-01-github-pages-custom-domains-https/) for GitHub pages. :dancer:

**Comment by Echolon** [comment]
>  Do you want to submit a PR for the fonts?
Yes, unfortunately I don't how to this - I'm just always wondering why Google provides this service? I mean, they somehow get information out of it or? - But I cant really say something about it!

> GitHub now supports Let's encrypt certificates for GitHub pages

Guess who pushed them to do so? :wink:
